app/controllers/concerns/two_factor_authenticatable_methods.rb

```diff
@@ -321,9 +321,4 @@ def presenter_for_two_factor_authentication_method
   def phone_configuration
     MfaContext.new(current_user).phone_configuration(user_session[:phone_id])
   end
-
-  def masked_number(number)
-    return '' if number.blank?
-    "***-***-#{number[-4..-1]}"
-  end
```

Comment:
I might pull this `masked_number` refactor into a separate PR just to keep the size on this one down

Reply:
https://github.com/18F/identity-idp/pull/5896